platforms/corda-ent/charts/pki-node/templates/job.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.nodeName }}-pki
+  namespace: {{ .Values.metadata.namespace }}
+  labels:
+    app: {{ .Values.nodeName }}-pki
+    app.kubernetes.io/name: {{ .Values.nodeName }}-pki
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    {{- include "labels.custom" . | nindent 2 }}
+spec:
+    backoffLimit: 10
+    template:
+      metadata:
+        labels:
+          app: {{ .Values.nodeName }}-pki
+          app.kubernetes.io/name: {{ .Values.nodeName }}-pki
+          app.kubernetes.io/instance: {{ .Release.Name }}
+      spec:
+        restartPolicy: "OnFailure"
+        serviceAccountName: {{ $.Values.vault.serviceaccountname }}
+        hostname: {{ .Values.nodeName }}
+        securityContext:
+          fsGroup: 1000
+        containers:
+        - name: pki
+          image: "{{ .Values.image.dockerImagePki.name }}:{{ .Values.image.dockerImagePki.tag }}"
+          env:
+            - name: ACCEPT_LICENSE
+              value: {{ .Values.acceptLicense }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/bin/bash", "-c"]
+          args:
+          - |-
+            rm -r DATA/done.txt
+            mkdir  DATA/node
+            time java -Xmx{{ .Values.cordaJarMx }}G -jar {{ .Values.jarPath }}/pkitool.jar --config-file {{ .Values.configPath }}/pki.conf --ignore-missing-crl
+            #creating a dummy file to perform check if last line is executed or not.
+            touch DATA/done.txt
```

Comment:
Is this required?

Reply:
Yes as this will be checked in next container.